src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2038,6 +2038,9 @@ private function addUidSection(ArrayNodeDefinition $rootNode, callable $enableIf
                         ->scalarNode('time_based_uuid_node')
                             ->cannotBeEmpty()
                         ->end()
+                        ->arrayNode('argument_value_resolver')
```

Comment:
Is there any downside about registering it unconditionally? Why this config?

Reply:
Yes, if a project already has a custom argument value resolver for uids, this new one will be used instead, especially since it has a high priority. Can we take the risk event if it's not very likely to exist?
I don't really know what's our policy for this kind of potential problem, I'm fine with both options.

Reply:
I'd vote for removing this config, we usually don't care about this (e.g serializer normalizers) ; it'll be rather easy to spot and fix in userland, which would already have a high priority to pass before the `RequestAttributeValueResolver`. We didn't add such a config for the `BackedEnumValueResolver` for instance. But perhaps the priority should be lower than `150`. Same as `BackedEnumValueResolver`, i.e `105` ?

Reply:
Fine to me 👍🏻